opsdroid/loader.py

```diff
@@ -177,20 +177,28 @@ def build_module_install_path(self, config):
         return os.path.join(self.modules_directory, config["type"], config["name"])
 
     @staticmethod
-    def git_clone(git_url, install_path, branch):
+    def git_clone(git_url, install_path, branch, key_path=None):
         """Clone a git repo to a location and wait for finish.
 
         Args:
             git_url: The url to the git repository
             install_path: Location where the git repository will be cloned
             branch: The branch to be cloned
+            key_path: SSH Key for git repository
 
         """
+        git_env = os.environ.copy()
+        if key_path:
+            git_env[
```

Comment:
This is the line that coverage is complaining about, I think in a test you could just `self.assertEqual` that `os.environ["GIT_SSH_COMMAND"]` is equal to that string (with the key path in). 

After that I believe coverage will go back up to 100% 😄 👍